clients/apple/Shared/Views/ManageSubscription.swift

```diff
@@ -0,0 +1,118 @@
+import SwiftUI
+import AlertToast
+
+struct ManageSubscription: View {    
+    @EnvironmentObject var billing: BillingService
+    @EnvironmentObject var settings: SettingsService
+    
+    @Environment(\.presentationMode) var presentationMode
+        
+    @State var isLoading = false
+    @State var error = false
+    
+    var body: some View {
+        VStack(alignment: .leading) {
+            VStack (alignment: .leading) {
+                Text("Current Usage:")
+                ColorProgressBar(value: settings.usageProgress)
+            }
+            .padding(.vertical)
+                
+            switch settings.tier {
+            case .Trial: trial
+            case .Premium: trial
+            case .Unknown: trial
+            }
+            
+            Text("Expand your storage to **30GB** for just **\(billing.maybeMonthlySubscription!.displayPrice)** a month.")
```

Comment:
```suggestion
            Text("Expand your storage to **30GB** for **\(billing.maybeMonthlySubscription!.displayPrice)** a month.")
```

What language are we using elsewhere?